pack.ps1

```diff
@@ -63,11 +63,21 @@ Write-Host "releaseVersion: " $releaseVersion;
 Write-Host "nugetPreview:   " $nugetPreview;
 
 CreateArtifactsFolder($artifactsRoot);
-foreach($project in $buildConfiguration.SelectNodes("root/projects/src/project"))
+
+foreach ($project in $buildConfiguration.SelectNodes("root/projects/src/project"))
 {
-	$name = $project.name;
-	Write-Host ">>> Start-Process -wait -NoNewWindow $dotnetexe 'pack' --no-build --no-restore -nodereuse:false -c $buildType -o $artifactsRoot -v q -s $root\src\$name\$name.csproj"
-	Start-Process -Wait -PassThru -NoNewWindow $dotnetexe "pack --no-build --no-restore -nodereuse:false -c $buildType -o $artifactsRoot -v q -s $root\src\$name\$name.csproj"
+    $name = $project.name
+    $projectPath = "$root\src\$name\$name.csproj"
```

Comment:
this won't work on linux/mac right?

How about [System.IO.Path]::Combine instead?

Reply:
My understanding is this script is not used by Wilson Linux builds.

Reply:
Updated to use Path.Combine.